### PR TITLE
refactor: mark AsteriskAsteriskEquals as assignment operator

### PR DESCRIPTION
**Pull request by armano2**
This PR adds AsteriskAsteriskEquals `**=` to assigmentOperators operator

### CODE REVIEW

File: src/node-utils.ts [hunk 1]
```diff
@@ -30,72 +32,73 @@ const ASSIGNMENT_OPERATORS = [
...
-TOKEN_TO_TEXT[SyntaxKind.AmpersandEqualsToken] = '&=';
-TOKEN_TO_TEXT[SyntaxKind.BarEqualsToken] = '|=';
-TOKEN_TO_TEXT[SyntaxKind.CaretEqualsToken] = '^=';
-TOKEN_TO_TEXT[SyntaxKind.AtToken] = '@';
-TOKEN_TO_TEXT[SyntaxKind.InKeyword] = 'in';
-TOKEN_TO_TEXT[SyntaxKind.UniqueKeyword] = 'unique';
-TOKEN_TO_TEXT[SyntaxKind.KeyOfKeyword] = 'keyof';
-TOKEN_TO_TEXT[SyntaxKind.NewKeyword] = 'new';
-TOKEN_TO_TEXT[SyntaxKind.ImportKeyword] = 'import';
+const TOKEN_TO_TEXT: { [k: number]: string } = {
```
Comment (j-f1) [inline src/node-utils.ts]: Could this be `SyntaxKind` instead?

```suggestion
const TOKEN_TO_TEXT: { [kind: SyntaxKind]: string } = {
```

Comment (armano2) [reply src/node-utils.ts]: sadly no, Typescript is not allowing only string and number as key

Comment (armano2) [reply src/node-utils.ts]: https://github.com/Microsoft/TypeScript/issues/2491

Comment (armano2) [reply src/node-utils.ts]: hmm, they added mappedTypes hmm https://github.com/Microsoft/TypeScript/pull/12114

Comment (armano2) [reply src/node-utils.ts]: @j-f1 
what do you think about:
```
{
  readonly [P in keyof typeof SyntaxKind]: string;
}
```
but i will have to define all of them

Comment (armano2) [reply src/node-utils.ts]: ok i updated it to:
```ts
const TOKEN_TO_TEXT: { readonly [P in keyof typeof SyntaxKind]?: string } = {
```

**Comment by j-f1** [comment]
Can you add a test?

**Comment by armano2** [comment]
@j-f1 there is already one: https://github.com/JamesHenry/typescript-estree/blob/master/tests/fixtures/javascript/exponentiationOperators/exponential-operators.src.js

**Comment by j-f1** [comment]
@armano2 You should be able to just push new commits since since this repo uses squash-merging so all the commits will be combined when the PR is merged.

**Comment by j-f1** [comment]
@armano2 It looks like you just have to follow the guidelines from [this package](https://www.npmjs.com/package/@commitlint/config-conventional).

**Pull request merged**